.github/workflows/test.yml

```diff
@@ -30,9 +30,14 @@ jobs:
           sudo apt-get install -y libfuse-dev
 
       - name: Install Python dependencies
-        run: |
-          test/ci-install.sh
+        if: ${{ matrix.python-version != '3.11' }}
+        run: pip install pytest "sphinx<7.0" "Cython<3"
+
+      - name: Install Python dependencies
+        if: ${{ matrix.python-version == '3.11' }}
```

Comment:
Perhaps make Cython version another variable in the matrix, and test at least one Cython version with all the Pythons. Not saying that's likely to discover anything new but it should make it clearer whether the issue stems from Python 3.11 or Cython 3.

Reply:
done.